src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.utils.DukeException;
+import duke.utils.DukeFileHandler;
+import duke.utils.TaskList;
+import duke.utils.Ui;
+
+
+public class DoneCommand extends Command {
+    private final int entryDone;
+
+
+    public DoneCommand(int entryDone) {
+        this.entryDone = entryDone;
+    }
+
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, DukeFileHandler fileHandler) throws DukeException {
+        try {
+            ui.displayThis("Nice! I've marked this task as done: \n        " + tasks.done(entryDone));
```

Comment:
While non-essential, maybe you can consider adding this display string as a specific method in your UI class instead?

Adopting this strategy will help to shorten the code in your command classes as a simple call to maybe ui.showDoneMessage(entryDone); can be used instead of having the entire message displayed here. 

It will also be easier to modify your display messages if they all reside in the same UI class. Just a point to consider 😸  

Otherwise, it works fine as it is as well 👍🏻